test.py

```diff
@@ -0,0 +1,121 @@
+class Tourist(object):
+
+    def get_info(self):
+        name = str(input('Enter your name: '))
+        room = int(input('Enter the room you want to book: '))
+        nights = int(input('How many nights?: '))
+        self.name = name
+        self.room = room
+        self.nights = nights
+        self.tourist_info = {'name': self.name,
+                             'room': self.room,
+                             'nights': self.nights}
+
+        return self.tourist_info
+
+
+class Hotel(object):
+    # Cодержит данные о номерах отеля.
+    # Номера комнат, информацию о стоимости номера в сутки,
+    # а также методы изменения стоимости в зависимости от сезона года
+
+    prices_all = {"room_101": 15,
+                  "room_102": 20,
+                  "room_103": 25,
+                  "room_104": 30}
+
+    def change_base_price(self):
+        """изменение стоимости любого из номеров"""
+
+        room = str(input('Enter number of room: '))
+        price = int(input('Enter new price: '))
+
+        for key in self.prices_all.keys():
+            if key[-3:] == room:
```

Comment:
maybe it would be better to use 101 102 etc as room numbers to avoid all these troubles?